scripts/e2e/hack/job.yml

```diff
@@ -11,7 +11,8 @@ spec:
         key: node-role.kubernetes.io/master
       containers:
       - name: cluster-api-provider-vsphere-ci
-        image: luoh/cluster-api-provider-vsphere-prow-ci:latest
+        image: gcr.io/cnx-cluster-api/cluster-api-provider-vsphere-ci-debug:debug
+        imagePullPolicy: Never
```

Comment:
Why push a debug version (above) if you never pull it?

Reply:
make push is optional only when the minikube cluster is on same machine as docker image been build, in other cases, developer can still push and pull ci debug image.